src/components/Chatbot/Chatbot.css

```
@@ -0,0 +1,266 @@
+  border-radius: 4px;
+}
+
+/* ── Messages ─────────────────────────────────────────────────────────────── */
+.chatbot-bot-msg,
+.chatbot-user-msg {
+  padding: 8px 12px;
+  margin: 3px 0;
+  max-width: 82%;
+  font-size: 0.875rem;
+  line-height: 1.45;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)